cli/tests/pcluster/validators/test_ebs_validators.py

```diff
@@ -94,30 +94,24 @@ def test_ebs_volume_iops_validators(volume_type, volume_size, volume_iops, expec
         ("standard", 15, None),
         ("standard", 0, "The size of standard volumes must be at least 1 GiB"),
         ("standard", 1025, "The size of standard volumes can not exceed 1024 GiB"),
-        ("standard", None, None),
```

Comment:
These were added in the previous PR. Now, the input parameter of the validator cannot be None, therefore I removed these tests